tests/test_rest_get_bucket_location.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats
+
+# Copyright 2025 Versity Software
+# This file is licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+load ./bats-support/load
+load ./bats-assert/load
+
+source ./tests/drivers/create_bucket/create_bucket_rest.sh
+source ./tests/drivers/get_bucket_location/get_bucket_location_rest.sh
+source ./tests/setup.sh
+
+@test "REST - GetBucketLocation - no bucket" {
```

Comment:
Are all the buckets created for testing here cleaned up?  I don't see a teardown function.